app/javascript/actionsPanel/initializeActionsPanelToggle.js

```diff
@@ -6,13 +6,9 @@ export function initializeActionsPanel(user, path) {
   `;
 
   const modActionsMenuIconHTML = `
-    <button class="mod-actions-menu-btn crayons-btn crayons-btn--icon-rounded crayons-btn--s">
-      <svg
-        xmlns="http://www.w3.org/2000/svg" width="54px" height="54px"
-        viewBox="-8 -8 40 40" class="crayons-icon actions-menu-svg" role="img"
-        aria-labelledby="d6cd43ffbad3fe639e2e95c901ee88c8">
-        <title id="d6cd43ffbad3fe639e2e95c901ee88c8">Moderation</title>
-        <path d="M3.783 2.826L12 1l8.217 1.826a1 1 0 01.783.976v9.987a6 6 0 01-2.672 4.992L12 23l-6.328-4.219A6 6 0 013 13.79V3.802a1 1 0 01.783-.976zM5 4.604v9.185a4 4 0 001.781 3.328L12 20.597l5.219-3.48A4 4 0 0019 13.79V4.604L12 3.05 5 4.604zM13 10h3l-5 7v-5H8l5-7v5z"></path>
+    <button class="mod-actions-menu-btn p-4 fixed right-3 bottom-9 m:bottom-7 m:right-7 z-elevate crayons-btn crayons-btn--icon-rounded">
+      <svg width="32" height="32" class="crayons-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+        <path d="M3.783 2.826L12 1l8.217 1.826a1 1 0 01.783.976v9.987a6 6 0 01-2.672 4.992L12 23l-6.328-4.219A6 6 0 013 13.79V3.802a1 1 0 01.783-.976zM5 4.604v9.185a4 4 0 001.781 3.328L12 20.597l5.219-3.48A4 4 0 0019 13.79V4.604L12 3.05 5 4.604zM13 10h3l-5 7v-5H8l5-7v5z"/>
```

Comment:
Is there a reason we're removing the title/aria-labelledby?

Reply:
Mistake. I'll bring these back.